src/backend/app/db/db_models.py

```diff
@@ -116,6 +116,7 @@ class DbProject(Base):
     organisation = relationship(DbOrganisation, backref="projects")
 
     # flight params
+    overlap = cast(float, Column(Float, nullable=True))  # in cm_px
     gsd = cast(float, Column(Float, nullable=True))  # in cm_px
```

Comment:
try to be precise in the namings for clarity (including units) - I used the names in did in the suggested schema for this purpose:
- ground_sample_distance_cm_px
- overlap_percent
- etc

Reply:
Someone looking at the database in the future will probably guess the units for `gsd` (if they know what it stands for).

By it might be harder to know in the Drones table that `weight` could be kg, american lbs, etc...